### PR TITLE
[show] Fixed cases quatre-ving-dix un, soixante-dix un

### PR DESCRIPTION
**Pull request by rtxm**

### CODE REVIEW

**Comment by rtxm** [comment]
**No Ready yet**
breaks "nonante"

**Comment by rtxm** [comment]
This is the definitive version.
We are now taking linguistic incompatibilities into account, not only the logical ones.

File: src/lang/fr/mod.rs [hunk 1]
```diff
@@ -32,13 +34,28 @@ impl French {
     }
 }
 
+bitflags! {
+    /// Words that can be temporarily banned because of linguistic features.
+    ///(logical, numerical feature inconsistencies are already taken care of by DigitString)
+    struct Banned: u64 {
```
Comment (rtxm) [inline src/lang/fr/mod.rs:40]: Not happy with the name. If you have some better proposals, I'm hearing you!

Comment (merwan) [reply src/lang/fr/mod.rs:40]: A few suggestions:
- Discarded
- Ignored
- Blocked
- Dropped
- Outliers

**Review by merwan — APPROVED** [review]
Tests pass, so I suppose it's working well!

**Pull request merged**